include/picongpu/simulation_defines/param/isaac.param

```diff
@@ -17,27 +17,37 @@
  * If not, see <http://www.gnu.org/licenses/>.
  */
 
+/** @file
+ *
+ * Definition of native fields and density fields of particles which shall be
+ * visualized with ISAAC.
+ *
+ */
+
 #pragma once
 
 namespace picongpu
 {
 namespace isaacP
 {
 
-    // Native fields to visualize
+    /** Compile time sequence of native fields of PIConGPU which shall be
```

Comment:
Maybe: `Intermediate list of native ...`

I would skip the information that this is a compile time list.